src/libs/actions/IOU.ts

```diff
@@ -8598,12 +8608,23 @@ function getPayMoneyRequestParams(
     );
 
     if (iouReport?.policyID) {
+        const prevLastUsedPaymentMethod = (lastUsedPaymentMethods?.[iouReport.policyID] as OnyxTypes.LastPaymentMethodType)?.lastUsed?.name;
+        const usedPaymentOption = paymentPolicyID ?? paymentMethodType;
+
+        const optimisticLastPaymentMethod = {
+            [iouReport.policyID]: {
+                ...(isInvoiceReport ? {invoice: {name: paymentMethodType, bankAccountID}} : {}),
+                ...(iouReport.type ? {[iouReport.type]: {name: usedPaymentOption}} : {}),
```

Comment:
This change overrides the above line:
```javascript
...(isInvoiceReport ? {invoice: {name: paymentMethodType, bankAccountID}} : {}),
```

@getusha This makes a bug where we couldn't find the last bank account